<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle"><string>Multi NDI Recorder</string></property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="controlsLayout">
      <item><widget class="QLabel" name="label"><property name="text"><string>Sources</string></property></widget></item>
      <item><widget class="QSpinBox" name="sourceCountSpin"><property name="minimum"><number>1</number></property><property name="maximum"><number>10</number></property></widget></item>
      <item><widget class="QPushButton" name="startAllButton"><property name="text"><string>Start All</string></property></widget></item>
      <item><widget class="QPushButton" name="pauseAllButton"><property name="text"><string>Pause All</string></property></widget></item>
      <item><widget class="QPushButton" name="stopAllButton"><property name="text"><string>Stop All</string></property></widget></item>
      <item><widget class="QLabel" name="masterStatusLabel"><property name="text"><string>Active sources: 0</string></property></widget></item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <widget class="QWidget" name="tabRecorder">
       <attribute name="title"><string>Recorder</string></attribute>
       <layout class="QGridLayout" name="gridLayout"/>
      </widget>
      <widget class="QWidget" name="tabLibrary">
       <attribute name="title"><string>Library</string></attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item><widget class="QTableView" name="libraryTable"/></item>
        <item>
         <layout class="QHBoxLayout" name="libraryButtons">
          <item><widget class="QPushButton" name="openButton"><property name="text"><string>Open</string></property></widget></item>
          <item><widget class="QPushButton" name="revealButton"><property name="text"><string>Reveal in Explorer</string></property></widget></item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
   </item>
   </layout>
  </widget>
 </widget>
<resources/>
<connections/>
</ui>
